src/ui/server/api/keystone/APIM/index.ts

```diff
@@ -0,0 +1,31 @@
+import { ApolloResponse } from '../../../../types';
+import apollo from '../../../graphql/apollo';
+import getApimCisCountries from '../../../graphql/queries/APIM/CIS-countries';
+
+const APIM = {
+  getCisCountries: async () => {
+    try {
+      const response = (await apollo('GET', getApimCisCountries, {})) as ApolloResponse;
+
+      if (response.errors) {
+        console.error('GraphQL network error querying NEW API CALL %O', response.errors);
```

Comment:
should it be APIM getCisCountries instead of NEW API CALL? same for rest in this file

Reply:
Whoops, have updated